### ISSUE TITLE
Freeze

### ISSUE THREAD

**Issue by rajan-craveri** [kr15h/ofxPiMapper#160]
Hi, I'm testing your image on a Rspberry Pi B+, playing one your source abstract video, after one hour the system freeze, video is paused or locked.
I have only one surface quad.

**Comment by kr15h** [comment]
Can you share the video file?

**Comment by micron-visual** [comment]
It's one of the default abstract video files the colored waves

Il gio 18 apr 2019, 18:02 Krisjanis Rijnieks <notifications@github.com> ha
scritto:

**Comment by kr15h** [comment]
I see. Will try to replicate that. Does it happen also after reboot?

**Comment by micron-visual** [comment]
Yes I tried to reboot, the app worked well but after an hour more or less
it freezes again. I can't exit the app, all the buttons doesn't respond
when freeze.

Il ven 19 apr 2019, 09:36 Krisjanis Rijnieks <notifications@github.com> ha
scritto:
